gradlew.bat

```diff
@@ -64,14 +64,29 @@ echo location of your Java installation.
 
 goto fail
 
+:init
```

Comment:
If they're not needed, please revert changes in gradlew and gradlew.bat,

Reply:
thank you! fixed